crates/holochain/src/conductor/space.rs

```diff
@@ -544,7 +544,11 @@ impl Spaces {
                             [end],
                             |row| row.get(0),
                         )?;
-                        DatabaseResult::Ok(Some((hashes, start..=end)))
+                        DatabaseResult::Ok(Some((
+                            hashes,
+                            kitsune_p2p_types::Timestamp::from_micros(start.0)
```

Comment:
Then where we have an integration point from a Holochain type to a Kitsune type, I've done the conversions in code that needs to change again anyway